staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -165,6 +212,101 @@ func translateTopology(pv *v1.PersistentVolume, topologyKey string) error {
 	return nil
 }
 
+// isTopologyLabelGA checks if the topology label used in this PV is GA
+// and return the zone/region label used. If no labels are found, check
+// NodeAffinity to find the label
+func isTopologyLabelGA(pv *v1.PersistentVolume) (bool, string, string) {
+	_, zoneGA := pv.Labels[v1.LabelTopologyZone]
+	_, regionGA := pv.Labels[v1.LabelTopologyRegion]
+	if zoneGA || regionGA {
+		return true, v1.LabelTopologyZone, v1.LabelTopologyRegion
```

Comment:
Is it possible the label could be GA, but the NodeAffinity could be beta? (or vice versa) It may be safer to look for the NodeAffinity label first?

Reply:
Yeah... I thought about it and it can become messy. For example, what if there is a skew like this. I think you are right, we should check for NodeAffinity first to make sure.

Reply:
Done